Source/Engine/Level/Actors/AnimatedModel.h

```diff
@@ -236,6 +244,14 @@ class FLAXENGINE_API AnimatedModel : public ModelInstanceActor
     /// <param name="worldSpace">True if convert matrices into world-space, otherwise returned values will be in local-space of the actor.</param>
     API_FUNCTION() void GetNodeTransformation(const StringView& nodeName, API_PARAM(Out) Matrix& nodeTransformation, bool worldSpace = false) const;
 
+    /// <summary>
+    /// Gets the node final transformation for a series of nodes.
+    /// </summary>
+    /// <param name="modelBoneNodes">The series of nodes that will be returned</param>
+    /// <param name="worldSpace">True if convert matrices into world-space, otherwise returned values will be in local-space of the actor.</param>
+    /// <returns></returns>
+    API_FUNCTION() void GetNodeTransformation(API_PARAM(Out) Array<ModelBoneNode>& modelBoneNodes, bool worldSpace = false) const;
```

Comment:
`API_PARAM(Ref)` as it works both ways so bindings will properly convert input array and output result.